torch/testing/_internal/common_methods_invocations.py

```diff
@@ -238,6 +238,8 @@ def method_tests():
         ('erfc', uniform_scalar(requires_grad=True), NO_ARGS, 'scalar', (True,)),
         ('erfinv', torch.rand(S, S, S).clamp(-0.9, 0.9), NO_ARGS),
         ('erfinv', normal_scalar_clamp(-0.9, 0.9, requires_grad=True), NO_ARGS, 'scalar'),
+        ('gcd', (S, S, S), ((S, S, S),), '', (False,)),
```

Comment:
How do I change this so that `gcd` gets invoked only with integral arguments?

@mruberry

Reply:
You don't need to add gcd or lcm to common methods invocations, since it's mostly intended for testing gradients, which you're not dealing with.

Reply:
Okay. In that case, I'm ready for a review.